src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -270,15 +270,16 @@ export default class DetailsBody extends React.Component<Props> {
             <Sidebar>
               <SidebarHeading>
                 <span>{t('Alert Rule')}</span>
-                {incident?.alertRule?.status !== AlertRuleStatus.SNAPSHOT && (
-                  <SideHeaderLink
-                    to={{
-                      pathname: `/organizations/${params.orgId}/alerts/metric-rules/${incident?.projects[0]}/${incident?.alertRule?.id}/`,
-                    }}
-                  >
-                    {t('View Alert Rule')}
-                  </SideHeaderLink>
-                )}
+                {incident?.projects?.[0] &&
+                  incident?.alertRule?.status !== AlertRuleStatus.SNAPSHOT && (
+                    <SideHeaderLink
+                      to={{
+                        pathname: `/organizations/${params.orgId}/alerts/metric-rules/${incident?.projects[0]}/${incident?.alertRule?.id}/`,
+                      }}
+                    >
+                      {t('View Alert Rule')}
+                    </SideHeaderLink>
+                  )}
```

Comment:
does `<Link>` support a `disabled` prop? Can we use that instead of hiding it? We should try to avoid having these loading animations